### PR TITLE
Incremental jsonrpc request id

### LINKED ISSUE #1060: Fix id in jsonrpc.go
It uses `method` as `id`, Looks a bit weird

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1060

### CODE REVIEW

**Comment by vikmeup** [comment]
Can we have it incremental?

**Comment by vikmeup** [comment]
Because when you do batch request, you will have same ID, most likely

File: pkg/blockatlas/jsonrpc.go [hunk 1]
```diff
@@ -16,14 +21,14 @@ type (
 		JsonRpc string      `json:"jsonrpc"`
 		Method  string      `json:"method"`
 		Params  interface{} `json:"params,omitempty"`
-		Id      string      `json:"id,omitempty"`
+		Id      int64       `json:"id,omitempty"`
```
Comment (catenocrypt) [inline pkg/blockatlas/jsonrpc.go:24]: Overly strict. As RPC may return string ID, I would not limit to int (still generate numeric ID, but as string)

Comment (hewigovens) [reply pkg/blockatlas/jsonrpc.go:24]: Some upstream rpc doesn’t support string id

**Pull request merged**